test/swfs/as3-loader/events/loader-events.swf.trace

```diff
@@ -0,0 +1,36 @@
+constructed root1
+added root1
+added instance1
+added instance3
+exitFrame 1
+root loader: init 1293
+root loader: progress 1293
+root loader: complete 1293
+enterFrame 2
+run frame 2 in root1
+exitFrame 2
+child 1 loader: open 0
+child 1 loader: progress 0
+child 1 loader: progress 8289
+child 2 loader: open 0
+child 2 loader: progress 0
+child 2 loader: progress 8289
+enterFrame 3
+added instance5
+construct instance2, parent: [object Loader] (TODO: parent should be null at this point, and the previous ADDED not dispatched)
+added instance2
+added instance6
+construct instance4, parent: [object Loader] (TODO: parent should be null at this point, and the previous ADDED not dispatched)
```

Comment:
The parent being null here should only be true for root objects and not all timeline objects per se. Looks like Flash is adding a loader's content child AFTER its constructor got called. That explains why its parent is null and no ADDED events are fired at this point.